daemon/graphdriver/aufs/aufs.go

```diff
@@ -67,7 +72,7 @@ func init() {
 type Driver struct {
 	root       string
 	sync.Mutex // Protects concurrent modification to active
-	active     map[string]int
+	active     map[string]*data
```

Comment:
data needs to be a pointer because `referenceCount` is modified. But we can also avoid having a pointer by setting the value again.